### DISCUSSION TITLE
Is it possible to enable a default application name for pgbouncer connections?

### DISCUSSION THREAD

**Discussion by tristan957** [pgbouncer/pgbouncer#1093] [Q&A]
I'm trying to analyze the origin of connections in our production environment. It would help me out if I could see the application_name of pgbouncer connections in Postgres's `connection authorized` logs.

**Comment by JelteF** [comment]
PgBouncer normally forwards the application_name that the client connections use. So if you change the application names of the clients that connect to pgbouncer you should be able to distinguish them.

If that's not possible you could use [`application_name_add_host`](https://www.pgbouncer.org/config.html#application_name_add_host), to add an ip address to the application_name of the client and distinguish them that way.

There is also `connect_query`, which can often be used to set defaults for postgres settings. But because PgBouncer forwards the client application_name afterwards. But there's no way to force the application_name currently (feel free to open an issue/PR if you really need this).
